core/theme/dark.js

```diff
@@ -115,7 +116,7 @@ Blockly.Themes.Dark.categoryStyles = {
 
 // This style is still being fleshed out and may change.
 Blockly.Themes.Dark =
-    new Blockly.Theme(Blockly.Themes.Dark.defaultBlockStyles,
+    new Blockly.Theme('dark', Blockly.Themes.Dark.defaultBlockStyles,
```

Comment:
`.dark` is very vague.  I suggest that we either auto-append `'Theme'` to their name, or add it just for our built-in themes.

Reply:
Fair enough.

Reply:
Done.